src/js/libcs/OpDrawing.js

```diff
@@ -389,8 +391,8 @@ eval_helper.drawconic = function(aConic, modifs) {
     Render2D.preDrawCurve();
 
     var eps = 1e-14; //JRG Hab ih von 1e-16 runtergesetzt
-    var mat = aConic.matrix;
-    var origmat = aConic.matrix;
+    var mat = List.normalizeMax(aConic.matrix);
+    var origmat = JSON.parse(JSON.stringify(mat));
```

Comment:
You are right, the unnecessary detour was due to safety considerations. I'll change that.